### ISSUE TITLE
Default target language

### ISSUE THREAD

**Issue by Donkey-Doug** [sakusaku3939/DeepLAndroid#75]
Make it possible to select a default language to translate to. I don't know why, but mine is always set on german. I need to manually select my language every time (dutch).

**Comment by sakusaku3939** [comment]
Once you have changed the translation language and entered it, it will be set to that language from the next time you start the program.
It seems to work fine in my environment.

**Comment by Donkey-Doug** [comment]
That feature does not work for me. Always in german.

Android 10
LineageOS (without google play services)

**Comment by sakusaku3939** [comment]
I see...
I'll let you know again when I find a way to reproduce it.

**Comment by GeoGlobber** [comment]
_Tl;dr open main app and perform translation to desired default language_

Hi, I can add some knowledge on this issue and how I was able to "solve" it:

**How to reproduce it:** 
Reinstall the app and use the app in popup mode (by selecting from context menu), the default language is always German

**What does NOT work:**
-Changing the language in the popup, it will be still be German the next time
-Going to the main app and selecting a different target language
-Doing the above and restarting the app (by force quitting)

**What works:**
Going to the main app, selecting desired target language _and actually make a translation towards it_

In this way, the correcr target language is saved.

**Expected behavior:**
When you change the target language in the popup, it should store it like it does in the main app.

Hope this helps!:)

**Comment by Donkey-Doug** [comment]
> 

This worked! Thank you.

**Comment by sakusaku3939** [comment]
@GeoGlobber Thanks for the info!
The fix for this bug will be incorporated into the next version.
